data/json/items/ranged/throwing.json

```
@@ -131,8 +132,9 @@
     "color": "light_gray",
     "description": "A thin, flat knife made for throwing.  Its ineffective cutting edge and odd shape make it unsuitable for use as a tool.",
     "material": [ "steel" ],
-    "volume": "100 ml",
-    "weight": "460 g",
+    "volume": "56 ml",
+    "weight": "200 g",
+    "longest_side": "20 cm",
```

```suggestion
    "longest_side": "35 cm",
```
World Knife Throwing League (WKTL) requires that they are between 13.5 and 16 inches long. This length allows throwers of different heights and arm lengths to hit targets at distances of 10 and 15 feet.